7_react/react2/week1/theme/src/App.js

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import ThemeContainer from "./components/ThemeContainer";
+import ThemedButton from "./components/ThemedButton";
+import ThemedHeader from "./components/ThemedHeader";
+import ThemedText from "./components/ThemedText";
+import "./themes/lightTheme.css";
+import "./themes/darkTheme.css";
+import "./themes/hackerTheme.css";
+import { ThemeContext } from "./contexts/ThemeContext";
+
+export default function App() {
+  const [theme, setTheme] = useState("dark");
+
+  return (
+    <>
+      <ThemeContext.Provider value={theme}>
+        <div className={"page-container " + theme}>
+          <div className="title-container">
+            <h1 className={"title " + theme}>Welcome to React Masters</h1>
+            <p className={"subtitle " + theme}>
+              We are bringing you all the themes you need for your applications.
+            </p>
+          </div>
+
+          <div className="themes-section">
+            <ThemeContext.Provider value={"light"}>
+              <ThemeContainer
+                title="Light theme"
+                buttonText="Use light theme"
+                onButtonClick={(x) => setTheme("light")}
+              >
+                <img
+                  src={process.env.PUBLIC_URL + "sun.svg"}
+                  alt="light theme"
+                />
```

Comment:
When you want to include an image or icon from the public folder you have to use the url:
```
process.env.PUBLIC_URL + {image path inside public folder}
```